plugins/locator/nll/nll_locator.cpp

```diff
@@ -426,6 +444,9 @@ bool NLLocator::init(const Config::Config &config) {
 		try { prof.tablePath = env->absolutePath(config.getString(prefix + "tablePath")); }
 		catch ( ... ) {}
 
+		try { prof.stationNameFormat = env->absolutePath(config.getString(prefix + "stationNameFormat")); }
```

Comment:
Why `absolutePath`? Is that is filename?

Reply:
copy/paste  hits again. I will fix this thanks